### PR TITLE
CY-1908 Add command for getting the node_id

### PR DESCRIPTION
**Pull request by inbala**

### CODE REVIEW

File: cfy_manager/main.py [hunk 1]
```diff
@@ -379,6 +382,12 @@ def db_node_set_master(**kwargs):
         logger.info('There is no database cluster associated with this node.')
 
 
+def get_id():
```
Comment (tyacbovi) [inline cfy_manager/main.py]: So it will be cfy_manager node get_id?

File: cfy_manager/utils/common.py [hunk 2]
```diff
@@ -217,3 +219,21 @@ def is_all_in_one_manager():
         DATABASE_SERVICE in config[SERVICES_TO_INSTALL] and
         QUEUE_SERVICE in config[SERVICES_TO_INSTALL]
     )
+
+
+def get_status_reporter_id():
```
Comment (tyacbovi) [inline cfy_manager/utils/common.py]: Why not get_node_id?

File: cfy_manager/utils/common.py [hunk 3]
```diff
@@ -217,3 +219,21 @@ def is_all_in_one_manager():
...
+    if not os.path.exists(STATUS_REPORTER_CONFIGURATION_PATH):
+        raise InitializationError(
+            'Status reporter is not installed, path does not exist: {0}'
+            .format(STATUS_REPORTER_CONFIGURATION_PATH)
+        )
+
+    try:
+        reporter_config = yaml.safe_load(
+            sudo(['cat', STATUS_REPORTER_CONFIGURATION_PATH]).aggr_stdout
```
Comment (tyacbovi) [inline cfy_manager/utils/common.py]: I am not against opening a new util file, even for a single func

File: cfy_manager/main.py (hunk 1 above)
Comment (inbala) [reply cfy_manager/main.py]: Yes, `cfy_manager node get-id`

File: cfy_manager/utils/common.py (hunk 2 above)
Comment (inbala) [reply cfy_manager/utils/common.py]: Sure, I'll change it

**Pull request merged**